app/src/main/java/com/github/codetanzania/ui/fragment/SliderItemsFragment.java

```diff
@@ -84,11 +90,8 @@ public void onClick(View v) {
         return Open311ServicesUtil.cached(getActivity());
     }
 
-    /* the interface is enables implementation to respond to "Show All Items" event when the button
-     * to preview all items is clicked */
-    public interface OnShowOpen311ServicesList {
-        void showList();
+    private void initializeIssueCategoryPickerDialog() {
+        mIssueCategoryPickerDialog = new IssueCategoryPickerDialog(
+                (ArrayList<Open311Service>) mSliderModelItems, mOnSelectIssueCategory);
```

Comment:
I feel the same about this one - why initialize it before it is clicked? Make it lazy load instead...